### ISSUE TITLE
[Request] Libretro/Retroarch core

### ISSUE THREAD

**Issue by QcSamuel** [k4zmu2a/SpaceCadetPinball#212]
Feature request. See above
https://github.com/libretro/RetroArch

**Comment by k4zmu2a** [comment]
Seem unlikely with low benefit/effort, lowest priority.

**Comment by Brian151** [comment]
does it even make sense in the first place?
aren't cores for, yanno, emulating entire game/computer systems?
among other reasons to _definitely_ not pursue this.

**Comment by QcSamuel** [comment]
> does it even make sense in the first place? aren't cores for, yanno, emulating entire game/computer systems? among other reasons to _definitely_ not pursue this.
my request would make sense in that if there a libretro core there would be more systems that could play spacecadet than waiting on a sourceport and upstream support could be useful if a sourceport is abandonned

there a outrun (arcade game) engine core on retroarch https://github.com/libretro/cannonball

there also cores for doom 3, quake, doom 1-2, cave story and more so not only emulation

**Comment by QcSamuel** [comment]
for me it would be a way to play it on my iphone but i guess i could open another issue for just that and close this complicated request 😅

**Comment by Brian151** [comment]
a flat port to $target [in this case, iOS] would genuinely make more sense
and ugh... apple... likely few people willing to touch that one
good luck i guess

**Comment by QcSamuel** [comment]
closed as a ios port would make more sense and there not much benefit to port it to retroarch in the end
